src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -77,8 +77,27 @@ export const disabilityNameTitle = ({ formData }) => {
 };
 
 
+export const facilityDescription = ({ formData }) => {
+  return (
+    <p>Tell us about facilities where VA treated you for {formData.disability.diagnosticText}, <strong>after you got your disability rating</strong>.</p>
+  );
+};
+
+
+export const treatmentView = ({ formData }) => {
+  const { startTreatment, endTreatment, treatmentCenterName } = formData.treatment;
+  return (
+    <div>
+      <strong>{treatmentCenterName}</strong>
+      <p>{startTreatment} — {endTreatment}</p>
```

Comment:
Should this date be 12/12/2012 or 12-12-2012? The latter seems like a lot of dashes if used thus: 12-12-2012—12-12-2012

Reply:
Yeah, that's true. What's the convention elsewhere?

Reply:
<img width="480" alt="screen shot 2018-03-09 at 4 52 58 pm" src="https://user-images.githubusercontent.com/16051172/37236219-71852a44-23ba-11e8-96f7-201d343c61d2.png">